app/experimenter/templates/base.html

```diff
@@ -104,21 +104,20 @@ <h3 class="pt-2">
     {% endif %}
 
     {% if messages %}
-    <div class="alert-danger">
-      <div class="container">
-        <div class="row">
-          <div class="col pt-3 pb-1">
-            {% for message in messages %}
-              <p>
-                <span class="fas fa-info-circle"></span>
-                {{ message }}
-              </p>
-            {% endfor %}
+      <div class="alert-danger">
+        <div class="container">
+          <div class="row">
+            <div class="col pt-3 pb-1">
+              {% for message in messages %}
+                <p>
+                  <span class="fas fa-info-circle"></span>
+                  {{ message }}
+                </p>
+              {% endfor %}
+            </div>
           </div>
         </div>
       </div>
-
-    </div>
     {% endif %}
 
```

Comment:
This is a whitespace change...  from fixing indentation of the block after the `{% if messages %}`, the reason why this has a change at all is because in earlier iterations this PR, this code was altered for clone success message notifications. I've taken those success messages out now that it's obvious something is happening when you go to make the clone. Also, to make those messages work now that we use an ajax request and modal seems tricky and possibly not worth the effort. Django `messages` use the request/response cycle to display messages the user hasn't seen yet, and we wouldn't be able to use that with modal and ajax.